### PR TITLE
spanish localization

### PR DESCRIPTION
**Pull request by Aikyuichi**
- [ ] This change has been discussed with the author.
- [ ] It implements / fixes issue #.

---

**Description:**

### CODE REVIEW

File: iina/es.lproj/InspectorWindowController.strings [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+"Fqo-1c-3L1.ibShadowedLabels[2]" = "Archivo";
+
+/* Class = "NSSegmentedCell"; Fqo-1c-3L1.ibShadowedLabels[3] = "Status"; ObjectID = "Fqo-1c-3L1"; */
+"Fqo-1c-3L1.ibShadowedLabels[3]" = "Estado";
+
+/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "Go3-bS-2ev"; */
+"Go3-bS-2ev.title" = "Label";
+
+/* Class = "NSTextFieldCell"; title = "Driver:"; ObjectID = "I2E-6Z-Gnn"; */
+"I2E-6Z-Gnn.title" = "Driver:";
```
Comment (saagarjha) [inline iina/es.lproj/InspectorWindowController.strings]: This too.

File: iina/es.lproj/InspectorWindowController.strings [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+"f7i-wX-uxQ.title" = "Pista:";
+
+/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "fYh-pA-yDU"; */
+"fYh-pA-yDU.title" = "Label";
+
+/* Class = "NSTextFieldCell"; title = "Estimated Output FPS:"; ObjectID = "g8p-Ec-awO"; */
+"g8p-Ec-awO.title" = "FPS de salida estimados:";
+
+/* Class = "NSTextFieldCell"; title = "VIDEO"; ObjectID = "gDG-Eq-1Bc"; */
+"gDG-Eq-1Bc.title" = "VIDEO";
```
Comment (saagarjha) [inline iina/es.lproj/InspectorWindowController.strings]: Wouldn't this be "VÍDEO"?

File: iina/es.lproj/InspectorWindowController.strings [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+
+/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "iMk-qU-kVv"; */
+"iMk-qU-kVv.title" = "Label";
+
+/* Class = "NSTextFieldCell"; title = "A/V Sync Diff:"; ObjectID = "iVK-Ck-Lyq"; */
+"iVK-Ck-Lyq.title" = "A/V Sync Diff:";
```
Comment (saagarjha) [inline iina/es.lproj/InspectorWindowController.strings]: This too

File: iina/es.lproj/InspectorWindowController.strings [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+"nrG-IH-kFg.title" = "AUDIO";
+
+/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "qPy-MN-AEF"; */
+"qPy-MN-AEF.title" = "Label";
+
+/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "rcA-a7-rVn"; */
+"rcA-a7-rVn.title" = "Label";
+
+/* Class = "NSTextFieldCell"; title = "Total A/V Sync:"; ObjectID = "sbO-Cn-gpI"; */
+"sbO-Cn-gpI.title" = "Total A/V Sync:";
```
Comment (saagarjha) [inline iina/es.lproj/InspectorWindowController.strings]: This too

File: iina/es.lproj/KeyBinding.strings [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+"cmd.frame-back-step" = "Fotograma anterior";
+"cmd.ab-loop" = "Bucle A-B";
+"cmd.sub-seek" = "Buscar subs";
+"cmd.playlist-next" = "Siguiente";
+"cmd.playlist-prev" = "Anterior";
+"cmd.playlist-move" = "Mover elemento de la lista de reproducción";
+"cmd.playlist-clear" = "Borrar lista de reproducción";
+"cmd.playlist-remove" = "Eliminar actual";
+"cmd.playlist-shuffle" = "Aleatorio";
+"cmd.write-watch-later-config" = "Record play history";
```
Comment (saagarjha) [inline iina/es.lproj/KeyBinding.strings]: This too

File: iina/es.lproj/KeyBinding.strings [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+"cmd.loadlist" = "Cargar lista de reproducción";
+"cmd.run" = "Ejecutar";
+
+"iina.flip" = "Voltear";
+"iina.mirror" = "Reflejar";
+
+"iina.toggle-pip" = "Alternar Picture-in-Picture";
+"iina.open-file" = "Abrir archivo";
+"iina.open-url" = "Abrir URL";
+"iina.video-panel" = "Panel de video";
```
Comment (saagarjha) [inline iina/es.lproj/KeyBinding.strings]: Again, wouldn't this be vídeo?

File: iina/es.lproj/KeyBinding.strings [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+"minus" = "disminuir";
+"forward" = "adelantar";
+"backward" = "restroceder";
+"seek-to" = "to";
+
+"relative" = "s";
+"absolute" = "s";
+"absolute-percent" = "%";
+"relative-percent" = "%";
+"relative+exact" = "s (exact)";
```
Comment (saagarjha) [inline iina/es.lproj/KeyBinding.strings]: @lhc70000 Should exact be translated here? It is in some translations, but not in others.

File: iina/es.lproj/KeyBinding.strings [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+"opt.gamma" = "Gama";
+"opt.saturation" = "Saturación";
+"opt.deinterlace" = "Desentrelazar";
+
+"opt.audio" = "Pista de audio";
+"opt.volume" = "Volumen";
+"opt.mute" = "Silenciar";
+"opt.audio-delay" = "Retrasar audio";
+
+"opt.sub" = "Sub track";
```
Comment (saagarjha) [inline iina/es.lproj/KeyBinding.strings]: @lhc70000 Again, should this be translated?

File: iina/es.lproj/KeyBinding.strings [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+"opt.chapter" = "Capítulo";
+
+
+"read.set" = "Establecer {{property}} a {{value}}";
+"read.add" = "{{property}} {{value}}";
+"read.multiply" = "Multiplicar {{property}} por {{factor}}x";
+"read.cycle" = "Alternar {{property}}{{#dir_down}} Downwards{{/dir_down}}";
+"read.cycle-values" = "Alternar {{property}} entre ({{{values}}})";
+"read.seek" = "Seek {{#fwd}}forward{{/fwd}}{{#bwd}}backward{{/bwd}}{{#abs}}to{{/abs}} {{value}}{{#sec}}s{{/sec}}{{#per}}%{{/per}} {{#exact}}(exact){{/exact}}{{#kf}}(keyframe){{/kf}}";
```
Comment (saagarjha) [inline iina/es.lproj/KeyBinding.strings]: @lhc70000

File: iina/es.lproj/Localizable.strings [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+"general.cancel" = "Cancelar";
+"general.username" = "Nombre de usuario";
+"general.password" = "Contraseña";
+
+// PlaylistViewController.swift
+"playlist.playlist_cap" = "LISTA DE REPRODUCCIÓN";
+"playlist.chapter_cap" = "CAPÍTULOS";
+
+// MainWindowController.swift
+"main.buffering_indicator" = "Buffering... %d%%";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: Translate this please.

File: iina/es.lproj/Localizable.strings [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+"osd.sub_saved" = "Subtítulo guardado";
+"osd.network_error" = "Error de red";
+"osd.file_error" = "Error al leer el archivo";
+"osd.cannot_login" = "No puedo iniciar sesión";
+"osd.canceled" = "Cancelado";
+
+"update.title_update_found" = "Actualización encontrada";
+"update.title_no_update" = "Ninguna actualización";
+"update.title_error" = "Error";
+"update.update_found" = "¡Se ha detectado una nueva versión! IINA todavía está en alfa, por lo que se recomienda permanecer con la última versión. Por favor, descargue desde el sitio web o github:";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: GitHub

File: iina/es.lproj/Localizable.strings [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+"update.no_update" = "Está ejecutando la versión más reciente.";
+"update.check_failed" = "No se pudo comprobar las actualizaciones. Probablemente esté ejecutando una versión demasiado antigua. Por favor, consulte el sitio web o github.";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: GitHub

File: iina/es.lproj/Localizable.strings [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+"update.new_version" = "Nueva versión:";
+"update.dl_from_website" = "Ir al sitio web";
+"update.dl_from_github" = "Ir a Github";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: GitHub

File: iina/es.lproj/KeyBinding.strings [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+"cmd.screenshot" = "Captura de pantalla";
+"cmd.screenshot-to-file" = "Screenshot to file";
+"cmd.loadfile" = "Carga Archivo";
+"cmd.loadlist" = "Cargar lista de reproducción";
+"cmd.run" = "Ejecutar";
+
+"iina.flip" = "Voltear";
+"iina.mirror" = "Reflejar";
+
+"iina.toggle-pip" = "Alternar Picture-in-Picture";
```
Comment (saagarjha) [inline iina/es.lproj/KeyBinding.strings]: We're going with "picture in picture" as per #691. Would you mind changing it?

File: iina/es.lproj/MainMenu.strings [hunk 15]
```diff
@@ -0,0 +1,387 @@
...
+"lYN-0x-lzT.title" = "Audio";
+
+/* Class = "NSMenuItem"; title = "Go To Screenshot Folder"; ObjectID = "lj0-Oo-7nB"; */
+"lj0-Oo-7nB.title" = "Ir a la carpeta de capturas de pantalla";
+
+/* Class = "NSMenuItem"; title = "Second Subtitle"; ObjectID = "mGw-jm-5x2"; */
+"mGw-jm-5x2.title" = "Segundo subtítulo";
+
+/* Class = "NSMenuItem"; title = "Enter Picture in Picture"; ObjectID = "mND-N8-z6o"; */
+"mND-N8-z6o.title" = "Ver en Picture in Picture";
```
Comment (saagarjha) [inline iina/es.lproj/MainMenu.strings]: Picture in Picture

File: iina/es.lproj/Localizable.strings [hunk 16]
```diff
@@ -0,0 +1,225 @@
...
+"preference.scripts" = "Scripts";
+"preference.not_logged_in" = "Sin iniciar sesión";
+"preference.logged_in_as" = "Conectado como %@";
+
+"menu.volume"= "Volumen: %d";
+"menu.audio_delay" = "Retraso de audio: %.2fs";
+"menu.sub_delay" = "Retraso de subtítulos: %.2fs";
+"menu.fullscreen" = "Ver a pantalla completa";
+"menu.exit_fullscreen" = "Salir de pantalla completa";
+"menu.pip" = "Ver en Picture-in-Picture";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: Picture in Picture

File: iina/es.lproj/Localizable.strings [hunk 17]
```diff
@@ -0,0 +1,225 @@
...
+"menu.exit_pip" = "Salir de Picture-in-Picture";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: Ditto

File: iina/es.lproj/Localizable.strings [hunk 18]
```diff
@@ -0,0 +1,225 @@
...
+"pl_menu.remove" = "Eliminar";
+"pl_menu.remove_multi" = "Eliminar seleccionados";
+"pl_menu.delete" = "Mover a la Papelera";
+"pl_menu.delete_multi" = "Mover los archivos seleccionados a la Papelera";
+"pl_menu.delete_after_play" = "Mover a la Papelera después de reproducir";
+"pl_menu.delete_after_play_multi" = "Mover los archivos seleccionados a la Papelera después de reproducir";
+"pl_menu.reveal_in_finder" = "Mostrar en el Finder";
+"pl_menu.add_item" = "Añadir Elemento...";
+"pl_menu.clear_playlist" = "Borrar la lista de reproducción";
+"pl_menu.matched_sub" = "Matched %d Subtitle(s)";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: Translate please.

File: iina/es.lproj/Localizable.strings [hunk 19]
```diff
@@ -0,0 +1,225 @@
...
+
+"touchbar.ahead_15" = "Adelantar 15s";
+"touchbar.ahead_30" = "Adelantar 30s";
+"touchbar.back_15" = "Retroceder 15s";
+"touchbar.back_30" = "Retroceder 30s";
+"touchbar.next_video" = "Siguiente Video";
+"touchbar.prev_video" = "Video anterior";
+"touchbar.increase_volume" = "Volumen +";
+"touchbar.decrease_volume" = "Volumen -";
+"touchbar.rewind" = "Rewind";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: Translate

File: iina/es.lproj/Localizable.strings [hunk 20]
```diff
@@ -0,0 +1,225 @@
...
+"touchbar.fast_forward" = "Fast Forward";
```
Comment (saagarjha) [inline iina/es.lproj/Localizable.strings]: Ditto

File: iina/es.lproj/MainMenu.strings [hunk 21]
```diff
@@ -0,0 +1,387 @@
...
+"8y7-Cy-lKT.title" = "Panel de configuración rápida";
+
+/* Class = "NSMenuItem"; title = "<None>"; ObjectID = "9pa-9C-ueQ"; */
+"9pa-9C-ueQ.title" = "<Ninguno>";
+
+/* Class = "NSMenuItem"; title = "Save Downloaded Subtitle"; ObjectID = "9zk-co-roT"; */
+"9zk-co-roT.title" = "Guardar los subtítulos descargados";
+
+/* Class = "NSMenuItem"; title = "Github"; ObjectID = "A9w-iE-gVg"; */
+"A9w-iE-gVg.title" = "Github";
```
Comment (saagarjha) [inline iina/es.lproj/MainMenu.strings]: GitHub

File: iina/es.lproj/PrefSubViewController.strings [hunk 22]
```diff
@@ -0,0 +1,174 @@
...
+"4EG-eE-S3y.title" = "Posición vertical:";
+
+/* Class = "NSMenuItem"; title = "Shooter.cn"; ObjectID = "5Bo-HI-z0D"; */
+"5Bo-HI-z0D.title" = "Shooter.cn";
+
+/* Class = "NSBox"; title = "Border"; ObjectID = "5CZ-8l-KYJ"; */
+"5CZ-8l-KYJ.title" = "Borde";
+
+/* Class = "NSTextFieldCell"; title = "Please enter a comma-separated list of strings."; ObjectID = "5eA-EB-Hco"; */
+"5eA-EB-Hco.title" = "Por favor, introduzca una lista de cadenas separadas por comas.";
```
Comment (saagarjha) [inline iina/es.lproj/PrefSubViewController.strings]: Not quite sure if a comma is needed here

File: iina/es.lproj/PrefSubViewController.strings [hunk 23]
```diff
@@ -0,0 +1,174 @@
...
+
+/* Class = "NSTextFieldCell"; title = " X:"; ObjectID = "7NC-Ih-gMB"; */
+"7NC-Ih-gMB.title" = " X:";
+
+/* Class = "NSTextFieldCell"; title = "Font spacing:"; ObjectID = "99d-UG-eDD"; */
+"99d-UG-eDD.title" = "Espaciado de la fuente:";
+
+/* Class = "NSButtonCell"; title = "Display subtitles in letter box in full screen"; ObjectID = "9kA-0S-Nl8"; */
+"9kA-0S-Nl8.title" = "Display subtitles in letter box in full screen";
```
Comment (saagarjha) [inline iina/es.lproj/PrefSubViewController.strings]: Translate please

File: iina/es.lproj/PrefSubViewController.strings [hunk 24]
```diff
@@ -0,0 +1,174 @@
...
+"A1r-kr-uDi.title" = "Escalar los subtítulos al tamaño de la ventana";
+
+/* Class = "NSTextFieldCell"; title = "strip"; ObjectID = "AnI-pG-XQo"; */
+"AnI-pG-XQo.title" = "Tira";
+
+/* Class = "NSTextFieldCell"; title = "sans-serif"; ObjectID = "Bjp-HJ-i7V"; */
+"Bjp-HJ-i7V.title" = "sans-serif";
+
+/* Class = "NSTextFieldCell"; title = "Opensubtitles account:"; ObjectID = "CKG-r9-NoN"; */
+"CKG-r9-NoN.title" = "Cuenta de Opensubtitles:";
```
Comment (saagarjha) [inline iina/es.lproj/PrefSubViewController.strings]: OpenSubtitles

File: iina/es.lproj/QuickSettingViewController.strings [hunk 25]
```diff
@@ -0,0 +1,342 @@
...
+"j9w-YU-EcC.title" = "2k";
+
+/* Class = "NSTabViewItem"; label = "Subtitle"; ObjectID = "jND-MZ-sKB"; */
+"jND-MZ-sKB.label" = "Subtítulo";
+
+/* Class = "NSTextFieldCell"; title = "0 dB"; ObjectID = "lBn-YS-jL7"; */
+"lBn-YS-jL7.title" = "0 dB";
+
+/* Class = "NSButtonCell"; title = "VIDEO"; ObjectID = "lQt-I0-jHO"; */
+"lQt-I0-jHO.title" = "VIDEO";
```
Comment (saagarjha) [inline iina/es.lproj/QuickSettingViewController.strings]: VÍDEO

**Comment by Aikyuichi** [comment]
Thanks for the feedback, I will take your suggestions into account

File: iina/es.lproj/InspectorWindowController.strings (hunk 1 above)
Comment (Aikyuichi) [inline iina/es.lproj/InspectorWindowController.strings]: ✔ translated

File: iina/es.lproj/KeyBinding.strings (hunk 14 above)
Comment (Aikyuichi) [inline iina/es.lproj/KeyBinding.strings]: Changed

File: iina/es.lproj/QuickSettingViewController.strings (hunk 25 above)
Comment (Aikyuichi) [inline iina/es.lproj/QuickSettingViewController.strings]: Changed

File: iina/es.lproj/MainMenu.strings (hunk 15 above)
Comment (Aikyuichi) [inline iina/es.lproj/MainMenu.strings]: Changed

File: iina/es.lproj/Localizable.strings (hunk 16 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Changed

File: iina/es.lproj/KeyBinding.strings (hunk 6 above)
Comment (Aikyuichi) [inline iina/es.lproj/KeyBinding.strings]: Changed

File: iina/es.lproj/InspectorWindowController.strings (hunk 2 above)
Comment (Aikyuichi) [inline iina/es.lproj/InspectorWindowController.strings]: Changed

File: iina/es.lproj/PrefSubViewController.strings (hunk 24 above)
Comment (Aikyuichi) [inline iina/es.lproj/PrefSubViewController.strings]: Changed

File: iina/es.lproj/MainMenu.strings (hunk 21 above)
Comment (Aikyuichi) [inline iina/es.lproj/MainMenu.strings]: Changed

File: iina/es.lproj/HistoryWindowController.strings [hunk 26]
```diff
@@ -0,0 +1,78 @@
...
+"F0z-JX-Cv5.title" = "Historial de Reproducción";
+
+/* Class = "NSMenuItem"; title = "Reveal in Finder"; ObjectID = "FPK-nx-hvF"; */
+"FPK-nx-hvF.title" = "Mostrar en el Finder";
+
+/* Class = "NSMenuItem"; title = "Search in"; ObjectID = "Gx5-gP-rzr"; */
+"Gx5-gP-rzr.title" = "Buscar en";
+
+/* Class = "NSTableColumn"; headerCell.title = "Played at"; ObjectID = "M7K-6H-OPw"; */
+"M7K-6H-OPw.headerCell.title" = "Played at";
```
Comment (Aikyuichi) [inline iina/es.lproj/HistoryWindowController.strings]: Translated

File: iina/es.lproj/Localizable.strings (hunk 11 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Changed

File: iina/es.lproj/Localizable.strings (hunk 12 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Changed

File: iina/es.lproj/Localizable.strings (hunk 13 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Changed

File: iina/es.lproj/Localizable.strings (hunk 17 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Changed

File: iina/es.lproj/Localizable.strings (hunk 18 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Translated

File: iina/es.lproj/Localizable.strings (hunk 19 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Translated

File: iina/es.lproj/Localizable.strings (hunk 20 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Translated

File: iina/es.lproj/PrefSubViewController.strings (hunk 23 above)
Comment (Aikyuichi) [inline iina/es.lproj/PrefSubViewController.strings]: Translated

File: iina/es.lproj/InspectorWindowController.strings (hunk 3 above)
Comment (Aikyuichi) [inline iina/es.lproj/InspectorWindowController.strings]: Translated

File: iina/es.lproj/InspectorWindowController.strings (hunk 4 above)
Comment (Aikyuichi) [inline iina/es.lproj/InspectorWindowController.strings]: Translated

File: iina/es.lproj/KeyBinding.strings (hunk 5 above)
Comment (Aikyuichi) [inline iina/es.lproj/KeyBinding.strings]: Translated

File: iina/es.lproj/Localizable.strings (hunk 10 above)
Comment (Aikyuichi) [inline iina/es.lproj/Localizable.strings]: Translated

File: iina/es.lproj/PrefSubViewController.strings [hunk 27]
```diff
@@ -0,0 +1,174 @@
...
+"5eA-EB-Hco.title" = "Por favor, introduzca una lista de cadenas separadas por comas.";
+
+/* Class = "NSTextFieldCell"; title = " X:"; ObjectID = "7NC-Ih-gMB"; */
+"7NC-Ih-gMB.title" = " X:";
+
+/* Class = "NSTextFieldCell"; title = "Font spacing:"; ObjectID = "99d-UG-eDD"; */
+"99d-UG-eDD.title" = "Espaciado de la fuente:";
+
+/* Class = "NSButtonCell"; title = "Display subtitles in letter box in full screen"; ObjectID = "9kA-0S-Nl8"; */
+"9kA-0S-Nl8.title" = "Mostrar los subtítulos en formato cinematográfico cuando esta en pantalla completa"; //Display subtitles in letter box in full screen
```
Comment (alejx) [inline iina/es.lproj/PrefSubViewController.strings]: Too long

File: iina/es.lproj/PrefUIViewController.strings [hunk 28]
```diff
@@ -0,0 +1,72 @@
...
+"mdD-if-Wqw.title" = "Ajustar al centro al arrastrar";
+
+/* Class = "NSMenuItem"; title = "Top"; ObjectID = "se2-DE-pJk"; */
+"se2-DE-pJk.title" = "En la parte superior";
+
+/* Class = "NSTextFieldCell"; title = "Text size:"; ObjectID = "suy-e8-j4i"; */
+"suy-e8-j4i.title" = "Tamaño del texto:";
+
+/* Class = "NSButtonCell"; title = "Resize to video size only when opening files manually"; ObjectID = "uD1-HB-46y"; */
+"uD1-HB-46y.title" = "Cambiar el tamaño al tamaño del vídeo sólo al abrir archivos manualmente";
```
Comment (alejx) [inline iina/es.lproj/PrefUIViewController.strings]: Too long

Comment (Aikyuichi) [inline iina/es.lproj/PrefUIViewController.strings]: Changed to "Ajustar al tamaño del vídeo al abrir archivos manualmente"

File: iina/es.lproj/PrefSubViewController.strings (hunk 27 above)
Comment (Aikyuichi) [inline iina/es.lproj/PrefSubViewController.strings]: Changed to "Mostrar subtítulos en formato cine en pantalla completa"

**Comment by alejx** [comment]
@Aikyuichi Looks much better, thank you!

**Comment by saagarjha** [comment]
@lhc70000 Mind if I squash the merge commits?

**Comment by alejx** [comment]
#826

**Pull request merged**